Dockerfile

```diff
@@ -178,6 +178,7 @@ RUN zypper addrepo http://download.opensuse.org/repositories/home:illuusio/openS
     /usr/share/emacs/ \
     /usr/share/xemacs/ \
     /usr/share/locale/*/LC_MESSAGES/*.[mp]o* \
+    /var/log/ \
```

Comment:
I've checked that it exists again at the end, and is roughly empty. ;-)
but didnt want to write that check into either Dockerfile or into the .travis.yml docker run command.
My rough thinking about that is at https://github.com/coala/docker-coala-base/issues/139